### PR TITLE
Docs: add doc for new variablesPanelMaxDepth config

### PR DESCRIPTION
**Pull request by LordSimal**
ref #846

### CODE REVIEW

File: docs/en/index.rst [hunk 1]
```diff
@@ -58,6 +58,12 @@ Configuration
 
 * ``DebugKit.ignoreAuthorization`` - Set to true to ignore Cake Authorization plugin for DebugKit requests. Disabled by default.
 
+* | ``DebugKit.variablesPanelMaxDepth`` - Defines how many levels of nested data should be shown in the variables tab. Default is 5.
+  | WARNING: Increasing the max depth level can lead to an out of memory error.
```
Comment (ADmad) [inline docs/en/index.rst]:
```suggestion
* ``DebugKit.variablesPanelMaxDepth`` - Defines how many levels of nested data should be shown in the variables tab. Default is 5.
  WARNING: Increasing the max depth level can lead to an out of memory error.
```

File: docs/en/index.rst [hunk 2]
```diff
@@ -58,6 +58,12 @@ Configuration
 
 * ``DebugKit.ignoreAuthorization`` - Set to true to ignore Cake Authorization plugin for DebugKit requests. Disabled by default.
 
+* ``DebugKit.variablesPanelMaxDepth`` - Defines how many levels of nested data should be shown in the variables tab. Default is 5.
+  WARNING: Increasing the max depth level can lead to an out of memory error.
```
Comment (ADmad) [inline docs/en/index.rst]:
```suggestion
  WARNING: Increasing the max depth level can lead to an out of memory error.::
```

**Comment by LordSimal** [comment]
I thought the `|` were necessary to have multiple lines in a bullet list 😅 
Still have to learn the RST syntax...

**Comment by ADmad** [comment]
RST is a weird beast, I still often make mistakes 😛.

**Pull request merged**